WalletWasabi.Documentation/FAQ.md

```diff
@@ -0,0 +1,178 @@
+
+### Why is the minimum mixing amount a weird number?
+
+The output value changes each round to ensure that you can que a coin and have it remix (mix over and over again - increasing the anonymity set, improving privacy). As a result the round mixing amount will often be a specific number which generally decreases as the rounds proceed, with a reset once a lower bound is reached. 
+
+# Post-Mix
+
+### What do I do now that I have mixed my coins?
+
+Wasabi is a very good wallet, and it is advisable to manage your funds using the wallet once you have mixed. If you are going to send your funds to another wallet (say, a mobile wallet for convenience) then there are a couple of important things to consider.
+- Unless the wallet has coin control you will merge your coins which damages privacy. 
+- Unless the wallet connects to your own full node you will leak information to the server supplying the blocks/filters.
+As a result, there are few wallets that are suitable. 
+
+Personally, I recommend sending your Zerolink Change to [Samourai Wallet](https://samouraiwallet.com/) or [Blockstream Green](https://blockstream.com/green/) for small-spends where you are less concerned about revealing any information. 
+
+### Can I recombine my mixed coins?
+
+It is advisable to limit the recombining of mixed coins because it can only decrease the privacy of said coins. This links all the consolidated UTXOs in one transaction, creating only one output, which then clearly controls all these funds. That said, if you combine less than 1 BTC it is less likely to reveal your pre-coinjoin transaction history. The potential issue comes when you spend that coin. Depending on what you do with the coin you might reduce the privacy of the resulting change (if you send half your coin to an exchange for example, as they will know that you own the coin change). As a result it is best not to recombine ALL your mixed change, though you may wish to recombine some coins if you are planning on hodling for many years as this will reduce the fees required to spend the coins later.
+
+### Am I safe to send my mixed coins to my hardware wallet?
+
+Most hardware wallets communicate with servers to provide you with your balance. This reveals your public key to the server, which damages your privacy - the hardware company can now theoretically link together all your addresses. As a result **it is not recommended** that you send your mixed coins to an address associated with your hardware wallet unless you are confident that you have set up your hardware wallet in a way that it does not communicate with a 3rd party server (see below). 
+
+You can however manage your hardware wallet with the Wasabi interface.
+
+### How can I set up my hardware wallet with Wasabi properly?
+
+You can currently use the following hardware wallets **with Wasabi directly**.
+- Coldcard
+- Trezor Model T
+- Ledger Nano S
```

Comment:
This requires maintenance. I'd say remove it. I am sure when I implement Trezor One I won't update this document for example. Furtermore everyone can see the supported hardware wallets when the user clicks on the hardware wallet tab.

Reply:
Simplified